packages/bpk-component-calendar/src/BpkCalendarNav.js

```diff
@@ -81,17 +84,20 @@ const BpkCalendarNav = props => {
         >
           <ArrowLeftIcon className={getClassName('bpk-calendar-nav__icon')} />
           <span className={getClassName('bpk-calendar-nav__text--hidden')}>
-            {formatMonth(addMonths(baseMonth, -1))}
+            {previousMonthLabel || formatMonth(prevMonth)}
```

Comment:
reusing [prevMonth from here](https://github.com/Skyscanner/backpack/pull/2132/files?file-filters%5B%5D=.js#diff-d4348e792a2389203dd012f08910249b4d0fb5c56ab4a319d3450468f65d7d9fR66) seems like it was duplicate code initially.

Using either previousMonthLabel or formatMonth(prevMonth) in order to keep the existing behaviour for those that don't have translations/custom text for their nudgers.